### PR TITLE
initial ts

### PR DESCRIPTION
**Pull request by jckhang**
Changes:

- [x] all types 
- [x] github workflow ci step
- [ ] check function (compares python & typescript api)
- [ ] all apis
- [x] publish as npm module
- [x] pr template
- [ ] readme

Links: 
- xyme-notebooks: https://github.com/Accern/xyme-notebooks/pull/20
- xyme-backend: https://github.com/Accern/xyme-backend/pull/116
- JIRA:
  * https://myaccern.atlassian.net/browse/TAD-2512
  * https://myaccern.atlassian.net/browse/TAD-2622
  * https://myaccern.atlassian.net/browse/TAD-2814
  * https://myaccern.atlassian.net/browse/TAD-2822

### CODE REVIEW

File: .ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,1189 @@
...
+   "source": [
+    "// const config: XYMEConfig = {\n",
+    "//     url: \"http://localhost:8080/\",\n",
+    "//     token: \"5173c3a07df24475a05d911977904b4b\",\n",
+    "//     namespace: \"default\"\n",
+    "// }\n",
+    "\n",
+    "const testConfig = {\n",
+    "    url: \"http://test-v3.accern.com/xyme\",\n",
+    "    token: \"b70ae4cc1cb94a65967e129004900a27\",\n",
```
Comment (JosuaKrause) [inline .ipynb_checkpoints/Untitled-checkpoint.ipynb:27]: you know that this repo is publicly visible, right? make sure to not include those files/information!

File: packages/typescript/accern_xyme.ts [hunk 2]
```diff
@@ -0,0 +1,1387 @@
...
+        });
+        this.nodeDefs = nodeDefs.info;
+        return nodeDefs.info;
+    }
+
+    public async createNewBlob(blobType: string): Promise<string> {
+        return await this.requestJSON<BlobInit>({
+            method: METHOD_POST,
+            path: '/blob_init',
+            addNamespace: false,
```
Comment (JosuaKrause) [inline packages/typescript/accern_xyme.ts:322]: how?

File: packages/typescript/accern_xyme.ts [hunk 3]
```diff
@@ -0,0 +1,1387 @@
...
+
+    public async createNewDag(
+        userName?: string,
+        dagName?: string,
+        index?: string
+    ) {
+        return await this.requestJSON<DagInit>({
+            method: METHOD_POST,
+            path: '/dag_init',
+            addNamespace: false,
```
Comment (JosuaKrause) [inline packages/typescript/accern_xyme.ts:337]: same here?

File: packages/python/accern_xyme/v3/accern_xyme.py [hunk 4]
```diff
@@ -2359,7 +2359,7 @@ def convert_model(self) -> ModelReleaseResponse:
             METHOD_POST, "/convert_model", {
                 "blob": self._uri,
                 "pipeline": self.get_pipeline().get_id(),
-            }
+            },
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/v3/accern_xyme.py:2362]:
```suggestion
            }))
```

File: packages/python/accern_xyme/v3/accern_xyme.py [hunk 5]
```diff
@@ -2359,7 +2359,7 @@ def convert_model(self) -> ModelReleaseResponse:
...
         ))
```
Comment (JosuaKrause) [inline packages/python/accern_xyme/v3/accern_xyme.py:2363]:
```suggestion
```

**Comment by JosuaKrause** [comment]
are there tickets for the unfinished items?

**Review by JosuaKrause — APPROVED** [review]
lgtm -- also the backend one

**Comment by jckhang** [comment]
remaining items go to https://myaccern.atlassian.net/browse/TAD-2916

**Pull request merged**